compiler/src/iree/compiler/Dialect/Flow/Transforms/ElementwiseOpFusion.cpp

```diff
@@ -82,6 +82,28 @@ void ElementwiseOpFusionPass::runOnOperation() {
     getOperation()->emitOpError("Failed to perform elementwise operations");
     return signalPassFailure();
   }
+
+  IRRewriter rewriter(context);
+  getOperation()->walk([&](linalg::GenericOp op) {
+    if (!linalg::isElementwise(op) || op.getNumResults() != 1)
```

Comment:
Would it make more sense to add it as a pattern to this pass https://github.com/iree-org/iree/blob/main/compiler/src/iree/compiler/Dialect/Flow/Transforms/FusionPreprocessing.cpp . It would apply repeatedly cause once the op has output map as identity things should be fine.

Reply:
I mis-track the PR..  Can @IanWood1 help pick up the PR?

Reply:
Yes! ill take a look